components/hab/src/exec.rs

```diff
@@ -13,15 +15,14 @@ use crate::{common::{self,
                               PackageInstall,
                               PackageTarget},
                     url::default_bldr_url,
-                    ChannelIdent}};
-use std::path::PathBuf;
-
-use crate::{error::{Error,
-                    Result},
+                    ChannelIdent},
             PRODUCT,
             VERSION};
+use retry::{delay,
+            retry};
+use std::path::PathBuf;
 
-const RETRY_LIMIT: u8 = 5;
+const RETRY_LIMIT: usize = 5;
```

Comment:
Just for my own edification, why change this?

Reply:
Because it's used as the `n` parameter to [`Iterator::take(self, n: usize)`](http://doc.rust-lang.org/1.36.0/core/iter/trait.Iterator.html#method.take)

Reply:
The best possible reason!